pkg/services/anonymous/anonimpl/impl.go

```diff
@@ -81,8 +89,9 @@ func (a *AnonDeviceService) TagDevice(ctx context.Context, httpReq *http.Request
 	}
 
 	anonDevice := &Device{
-		ip:        clientIPStr,
-		userAgent: httpReq.UserAgent(),
+		Kind:      kind,
+		IP:        clientIPStr,
+		UserAgent: httpReq.UserAgent(),
```

Comment:
can we add the timestamp here? that would be useful to display the "last seen at" time for a given device

Reply:
might also be an idea to change the name of this variable to make it clearer that it's a device, not necessarily anonymous

Reply:
Added and renamed. 

I think if we plan on keeping authed device stats on the long term, we should move this service separately